doc/source/conf.py

```diff
@@ -112,6 +112,26 @@ def setup(app):
 
 html_theme = 'pandas_sphinx_theme'
 
+html_theme_options = {
+    "external_links": [
+        {
+            'url': "https://pandas.pydata.org/pandas-docs/stable/",
+            "name": "Pandas Docs"
+        }
+    ],
+    "github_url": "https://github.com/pandas-dev/pandas-sphinx-theme",
+    "twitter_url": "https://twitter.com/pandas_dev",
+    "use_edit_page_button": True
```

Comment:
It would be great to get the edit on github button back into action. Do you know where it should appear on the page? I looked at the [docs preview](https://856-87849371-gh.circle-artifacts.com/0/html/amazon/step-zero-aws.html) but couldn't find it :(

Reply:
It's an unresolved issue: https://github.com/pandas-dev/pydata-bootstrap-sphinx-theme/issues/62
I think it'd be best to leave out this config change until the mentioned issue is resolved.

Reply:
@betatim @manics this was brought up in issue #1547. 
Regarding fixing the edit button, there is a temporary fix for it: changing `doc` to `doc/source`. I could add that to this PR if needed?

Thanks for the review!

Reply:
Looks there's no decision on #1547. If you want to suggest this change anyway could you do it in a separate PR, so it's easy to see the changes and revert them when the theme is fixed?

Reply:
@manics alright. Shall stick to fixing the other links for now. Will open a separate PR shortly.

Apologies for this, won't try to fix disparate issues in 1 PR now on.

Thanks!